contracts/modules/LoanClosingsShared.sol

```diff
@@ -37,6 +38,24 @@ contract LoanClosingsShared is
 
     enum CloseTypes { Deposit, Swap, Liquidation }
 
+    /** modifier for invariant check */
+    modifier iTokenSupplyUnchanged(bytes32 loanId) {
+        Loan storage loanLocal = loans[loanId];
+        LoanParams storage loanParamsLocal = loanParams[loanLocal.loanParamsId];
```

Comment:
loanParamsLocal are not required here anymore, no?

Reply:
removed